lib/crypto.js

```diff
@@ -83,6 +83,7 @@ exports.createHmac = exports.Hmac = Hmac;
 function Hmac(hmac, key, options) {
   if (!(this instanceof Hmac))
     return new Hmac(hmac, key, options);
+  this.key = key;
```

Comment:
None that I am aware of. I don't think Node makes any promises that Hmac keys are unextractable from memory after initialization, right?